<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoverManager</class>
 <widget class="QDialog" name="CoverManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cover Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QListWidget" name="artists">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="clear">
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="filter">
           <property name="hint" stdset="0">
            <string>Enter search terms here</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="view">
           <property name="text">
            <string>View</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::MenuButtonPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fetch">
           <property name="text">
            <string>Fetch Missing Covers</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="albums">
         <property name="alternatingRowColors">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>120</width>
           <height>120</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>true</bool>
         </property>
         <property name="resizeMode">
          <enum>QListView::Adjust</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="action_show_fullsize">
   <property name="text">
    <string>Show fullsize...</string>
   </property>
  </action>
  <action name="action_fetch">
   <property name="text">
    <string>Fetch automatically</string>
   </property>
  </action>
  <action name="action_choose_manual">
   <property name="text">
    <string>Choose manual cover...</string>
   </property>
  </action>
  <action name="action_unset_cover">
   <property name="text">
    <string>Unset cover</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/lineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>artists</tabstop>
  <tabstop>clear</tabstop>
  <tabstop>filter</tabstop>
  <tabstop>view</tabstop>
  <tabstop>fetch</tabstop>
  <tabstop>albums</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>filter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>329</x>
     <y>13</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clear</sender>
   <signal>clicked()</signal>
   <receiver>filter</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>13</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
